src/plugins/visualizations/public/plugin.ts

```diff
@@ -181,6 +208,29 @@ export class VisualizationsPlugin
     return {
       ...types,
       showNewVisModal,
+      getSavedVisualization: async (opts) => {
+        return getSavedVisualization(
+          {
+            search: data.search,
+            savedObjectsClient: core.savedObjects.client,
+            dataViews: data.dataViews,
+            spaces,
+            savedObjectsTagging: savedObjectsTaggingOss.getTaggingApi(),
+          },
+          opts
+        );
+      },
+      saveVisualization: async (savedVis, saveOptions) => {
+        return saveVisualization(savedVis, saveOptions, {
+          savedObjectsClient: core.savedObjects.client,
+          chrome: core.chrome,
+          overlays: core.overlays,
+          savedObjectsTagging: savedObjectsTaggingOss.getTaggingApi(),
```

Comment:
please keep in mind that `savedObjectsTaggingOss` plugin can be optional

Reply:
Agree, added `?`